Jenkinsfile

```diff
@@ -1033,7 +1035,8 @@ pipeline {
                     steps {
                         functionalTest inst_repos: el7_daos_repos(),
                                        inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                                                  ' ' + functional_rpms +
+                                                  ' python2-tabulate '
```

Comment:
same here. do we need this?

Reply:
n/a -- python2-tabulate is now in [daos.spec](https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec). 60c8ef19a03e1a7e5aaee2a59e7c2ab60f6d9362